src/include/ndpi_typedefs.h

```diff
@@ -1439,8 +1439,8 @@ struct ndpi_flow_struct {
     } dns;
 
     struct {
-      u_int8_t request_code;
       u_int8_t version;
+      u_int8_t mode;
```

Comment:
It seems reasonable.
We lost the `request_code` attribute for the old v2, but we now have correct `version` and `mode` for all 3 versions

Reply:
If someone is really interested in that attribute for v2, we can re-add it later